content/provide-an-endpoint/contents.lr

```diff
@@ -19,16 +19,10 @@ The [SpaceAPI validator](/validator) validates URLs or files. Please note that i
 ### HTTP and CORS
 
 Your server should
- * Be reachable via HTTPS to prevent websites having problems
-   with [mixed content
-   warnings](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content).
-
- * Set the `Access-Control-Allow-Origin`
-[CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) header to `*`
- * Set the
-   [`Content-Type`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type)
-   to `application/json; charset=utf-8` so that JavaScript based web
-   applications can query your endpoint directly.
+
+* Be reachable via HTTPS to prevent websites having problems with [mixed content warnings](https://developer.mozilla.org/en-US/docs/Web/Security/Mixed_content).
+* Set the [CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) `Access-Control-Allow-Origin` HTTP header to `*`, so that JavaScript based web applications can query your endpoint directly.
```

Comment:
Hmm actually all the 3 things are meant to enable JavaScript based web applications, right?

Reply:
I'm not aware of a content-type requirement for CORS. It's just best practice, but AFAIK doesn't break things of not done correctly.